bci_tester/data.py

```diff
@@ -363,7 +363,7 @@ def create_BCI(
         )
         LTSS_BASE_FIPS_CONTAINERS.append(
             create_BCI(
-                build_tag=f"{APP_CONTAINER_PREFIX}/ltss/sle15.3/bci-base-fips:{OS_CONTAINER_TAG}",
+                build_tag=f"{APP_CONTAINER_PREFIX}/ltss/sle15.3/bci-base-fips:15.3",
```

Comment:
is there a problem with OS_CONTAINER_TAG here?

Reply:
Nope, I hardcoded it before @dcermak suggested me to `export OS_VERSION=15.3`

It's removed now :)